database/ddl/sqlite/worker.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package sqlite
+
+const (
+	// CreateWorkerTable represents a query to
+	// create the workers table for Vela.
+	CreateWorkerTable = `
+CREATE TABLE
+IF NOT EXISTS
+Workers (
```

Comment:
This should be lowercase `workers` 👍 

```suggestion
workers (
```